### ISSUE TITLE
Subtitle freeze the progress, skips episode in middle of watching

### ISSUE THREAD

**Issue by Dirky14** [jellyfin/jellyfin-web#3702]
Q**Describe The Bug**

When I see a series with subtitles, sometimes the progress of the episode just freeze only with the subtitle, and can in some case skip to the next episode or returning to home if there isn't any other next episode. Quit and return to the episode does not change anything, only disabling the subtitle works (which is a bit problematic when I see japanese anime). I suppose this is a bug with jellyfin-web since I tried on other support, like Findroid, and I don't have this bug on these supports. I didn't have this bug in 10.7, I have it only after upgrading from 10.7 to 10.8.

**Steps To Reproduce**

1. Watch a series
2. Activate subtitles 
3. Wait
5. The episode will freeze or/and jellyfin-web will play the next episode/return to home

**Expected Behavior**

Have the series with subtitles working without having to disable/enable subtitles and return to homepage 5-6 times.
**Logs**

No trace on logs for this error

**Screenshots**

**System (please complete the following information):**
 - Platform: [e.g. Linux, Windows, iPhone, Tizen]
 - Browser: [e.g. Firefox, Chrome, Safari]
 - Jellyfin Version: [e.g. 10.6.0]
Linux, Firefox, Jellyfin 10.8.0. Jellyfin vesion on Docker, upgraded by re-creating the image on Portainer.
**Additional Context**

**Comment by Honken77** [comment]
I have the same issue. I can only see this in the logs when it happens:

`[2022-06-17 19:04:59.569 +01:00] [INF] [160] Jellyfin.Api.Helpers.TranscodingJobHelper: Deleting partial stream file(s) "/config/data/transcodes/d1e5cfa3594e129b9e823c2d173632c0.m3u8"`

`[2022-06-17 19:04:59.650 +01:00] [INF] [160] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Jellyfin Web" "10.8.0" playing "episode". Stopped at "1276441" ms`

And then it starts the new episode:

`[2022-06-17 19:04:59.683 +01:00] [INF] [139] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "user". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True`

`[2022-06-17 19:04:59.683 +01:00] [INF] [139] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="Anonymous Profile", Path="/data/tvshows/show/season/episode.mkv", AudioStreamIndex=null, SubtitleStreamIndex=null ) => ( PlayMethod=Transcode, TranscodeReason=SubtitleCodecNotSupported ) "media:/videos/6eddb71c-c9f8-df04-769a-da111da91fad/master.m3u8?MediaSourceId=6eddb71cc9f8df04769ada111da91fad&VideoCodec=h264,hevc,h264&AudioCodec=aac,mp3,ac3,eac3&AudioStreamIndex=1&SubtitleStreamIndex=2&VideoBitrate=139616000&AudioBitrate=384000&MaxFramerate=23.976025&api_key=<token>&SubtitleMethod=Encode&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=4dd240d617e93d11d96f39b6e9e54b00&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=False&h264-level=41&h264-videobitdepth=8&h264-profile=high&hevc-profile=main,main10&hevc-level=183&TranscodeReasons=SubtitleCodecNotSupported"`

Video: 1080p H264 SDR
Audio: English - DTS-HD MA - 5.1 - Default
Subtitles: SDH - English - PGSSUB

**Comment by Honken77** [comment]
I did some experimenting and switching the subtitles to "English - SUBRIP" stopped the crashes for me (I got some subtitles from Subscene and uploaded into the episodes).

**Comment by Dirky14** [comment]
I also had PGSSUB on my anime so maybe it's subtitle problem. This problem doesn't appear on Jellyfin Media Player on Linux.

**Comment by methatronc** [comment]
Same happens to me with ASS subtitles, skips to next episode right in the middle of one..
Happens since I upgraded to 10.8 as well.

**Comment by vardecab** [comment]
+1

**Comment by nyanmisaka** [comment]
Sounds it's a duplicate of https://github.com/jellyfin/jellyfin/issues/6957 ?

**Comment by methatronc** [comment]
Indeed, however since 6957 was closed maybe linking this issue to https://github.com/jellyfin/jellyfin/issues/8273 which is still open.
